### PR TITLE
Added currency symbol

### LINKED ISSUE #40: I changed the currency name in my metamask but the currency name did not change in the app.

### PR DESCRIPTION
**Pull request by UsamaSaleem14**
Work in this pr

1. Added currency symbol based on the current chain connected

image

### CODE REVIEW

**Comment by TBomberman** [comment]
Can you put a screenshot?

File: src/configs/chainconfig.json [hunk 1]
```diff
@@ -0,0 +1,855 @@
+{
+    "1": "ETH",
```
Comment (TBomberman) [inline src/configs/chainconfig.json:2]: I was just expecting to use the top 5 most propular chains. Can you make it to just the top 5?

Comment (TBomberman) [reply src/configs/chainconfig.json:2]: That's fine.

File: src/hooks/useStageHandler.ts [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import { getDateDifferenceInSeconds } from "./../utils/dateTimeUtils";
 import { toast } from "react-hot-toast";
 import { useCallback, useEffect, useState } from "react";
-import { useAccount, useBalance, useSigner } from "wagmi";
+import { useAccount, useSigner } from "wagmi";
 import {
   getCampaign,
   getContractCampaign,
+  getNetworkName,
```
Comment (TBomberman) [inline src/hooks/useStageHandler.ts]: change name

File: src/utils/ContractUtils.ts [hunk 3]
```diff
@@ -112,5 +112,8 @@ export const checkLinkValidity = (link: string): boolean => {
 
 export const getNetworkName = (networkId: string): string => {
```
Comment (TBomberman) [inline src/utils/ContractUtils.ts]: get the chain id here instead of from parameter

Comment (UsamaSaleem14) [reply src/utils/ContractUtils.ts]: Done

File: src/hooks/useStageHandler.ts (hunk 2 above)
Comment (UsamaSaleem14) [reply src/hooks/useStageHandler.ts]: Done

**Comment by UsamaSaleem14** [comment]
> 

image

**Pull request merged**